package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitmovin-player-ui",
-  "version": "3.43.0",
```

Comment:
Weird that this change shows up when it was already part of https://github.com/bitmovin/bitmovin-player-ui/pull/530 🤔

Reply:
🙈